scripts/update-electron-builder.js

```diff
@@ -0,0 +1,26 @@
+const fs = require('fs');
+const path = require('path');
+const yaml = require('js-yaml');
+
+// Paths to the files
+const packageJsonPath = path.join(__dirname, '..', 'package.json');
+const electronBuilderYmlPath = path.join(__dirname, '..', 'electron-builder.yml');
+
+// Read package.json
+const packageJson = JSON.parse(fs.readFileSync(packageJsonPath, 'utf8'));
+
+// Extract the holochain version
+const holochainVersion = packageJson.binaries.holochain;
+
+// Read electron-builder.yml
+const electronBuilderYml = yaml.load(fs.readFileSync(electronBuilderYmlPath, 'utf8'));
+
+// Update the productName
+electronBuilderYml.productName = `Holochain Launcher (${holochainVersion})`;
```

Comment:
It should not be the actual holochain version but only the non-breaking digits (e.g. 0.4 in case of holochain version 0.4.0-dev.8). We may update the holochain version shipped with the launcher as long as it's a non-breaking change (due to bugfixes or improvements in holochain) and this should not change the name of launcher.

Reply:
100% I will fix this